.nvmrc

```diff
@@ -1 +1 @@
-6.4.0
+10.13.0
```

Comment:
I think there's a reason this has remained at 6, but I don't know what that reason is.

Reply:
Oddly, there are dependencies in the code already that fail `yarn install` unless running node >= 8.13.0. Then the latest danger-js pushes this to 10.13. I'll try an older version of danger, if possible.

Reply:
Here's the PR for the version bump, including the debugging steps performed: https://github.com/department-of-veterans-affairs/vets-json-schema/pull/481